clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -80,6 +82,18 @@
         }
     }
 
+    static public final class BytesSerde implements Serde<Bytes> {
+        @Override
+        public Serializer<Bytes> serializer() {
+            return new BytesSerializer();
```

Comment:
Why are we returning a new instance of `ByteSerializer` everytime we call `serializer`?  (I think I had this question before.)  Same for the deserializer part.